macros/HTMLRefTable.ejs

```diff
@@ -111,11 +111,12 @@ var tag, badge,
 // BUSINESS LOGIC
 // --------------
 
-// Get the relevant pages for HTML Elements based on the template input
+// Get the relevant pages for HTML Elements based on the template input; include only
+// the part of the title that's actually the element name
 HTMLDocPages.forEach(function (page) {
-    var p,
-        t = lowerMe(page.title.replace(RGXCleanTag, "")),
-        tags = page.tags.map(lowerMe);
+    var p;
+    var t = lowerMe(page.slug.split("/").pop(-1)).replace(RGXCleanTag, "");
```

Comment:
`replace(RGXCleanTag, "")` is not needed anymore, right?

Reply:
(and thus the declaration of RGXCleanTag neither)

Reply:
Good catch. You're right.